src/__test__/AssetMakeComponent.test.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import expect from 'expect';
+import { spy } from 'sinon';
```

Comment:
Why the need for this? There is a `spyOn` function in jest

Reply:
It gave me quite a bit of errors before I ended up using sinon spy, but i ll give it another try.